--- src/main/java/xyz/pixelatedw/mineminenomi/entities/ThrownSpearEntity.java
+++ src/main/java/xyz/pixelatedw/mineminenomi/entities/ThrownSpearEntity.java
@@ -48,11 +48,11 @@
     }

     @Override
-    protected void defineSynchedData(SynchedEntityData.Builder builder) {
-        super.defineSynchedData(builder);
-        builder.define(ID_LOYALTY, (byte) 0);
-        builder.define(ID_FOIL, false);
-    }
+    protected void defineSynchedData(SynchedEntityData.Builder p_325925_) {
+        super.defineSynchedData(p_325925_);
+        p_325925_.define(ID_LOYALTY, (byte)0);
+        p_325925_.define(ID_FOIL, false);
+     }

     @Override
     public void tick() {
