--- src/main/java/xyz/pixelatedw/mineminenomi/init/ModDataComponents.java
+++ src/main/java/xyz/pixelatedw/mineminenomi/init/ModDataComponents.java
@@ -62,6 +62,9 @@
     public static final Supplier<DataComponentType<Integer>> ENERGY = DATA_COMPONENT_TYPES.register("energy", () ->
             DataComponentType.<Integer>builder().persistent(Codec.INT).networkSynchronized(ByteBufCodecs.VAR_INT).build());

+    public static final Supplier<DataComponentType<Integer>> COLA = DATA_COMPONENT_TYPES.register("cola", () ->
+            DataComponentType.<Integer>builder().persistent(Codec.INT).networkSynchronized(ByteBufCodecs.VAR_INT).build());
+
     public static void init(IEventBus eventBus) {
         DATA_COMPONENT_TYPES.register(eventBus);
     }
