--- src/main/java/xyz/pixelatedw/mineminenomi/api/abilities/Ability.java
+++ src/main/java/xyz/pixelatedw/mineminenomi/api/abilities/Ability.java
@@ -110,8 +110,8 @@
         return xyz.pixelatedw.mineminenomi.api.util.Result.success();
     }

-    public Result canUnlock(LivingEntity entity) {
-        return Result.success();
+    public xyz.pixelatedw.mineminenomi.api.util.Result canUnlock(LivingEntity entity) {
+        return xyz.pixelatedw.mineminenomi.api.util.Result.success();
     }

     protected long startTick = -1;
