--- src/main/java/xyz/pixelatedw/mineminenomi/items/weapons/ScissorsItem.java
+++ src/main/java/xyz/pixelatedw/mineminenomi/items/weapons/ScissorsItem.java
@@ -37,12 +37,19 @@
             boolean hasPassive = attackerStats.getActiveAbilities().contains(ModAbilities.REGISTRY.getKey(ModAbilities.KAGE_GIRI.get()).toString());

-            if (hasPassive && target instanceof Player targetPlayer) {
-                PlayerStats targetStats = targetPlayer.getData(ModDataAttachments.PLAYER_STATS);
-                if (targetStats.hasShadow()) {
-                    targetStats.setHasShadow(false);
-                    if (targetPlayer instanceof ServerPlayer serverPlayer) {
-                        ModNetworking.sendTo(new SUpdatePlayerStatsPacket(targetStats), serverPlayer);
+            if (hasPassive) {
+                if (target instanceof Player targetPlayer) {
+                    PlayerStats targetStats = targetPlayer.getData(ModDataAttachments.PLAYER_STATS);
+                    if (targetStats.hasShadow()) {
+                        targetStats.setHasShadow(false);
+                        if (targetPlayer instanceof ServerPlayer serverPlayer) {
+                            ModNetworking.sendTo(new SUpdatePlayerStatsPacket(targetStats), serverPlayer);
+                        }
+                        attackerPlayer.getInventory().placeItemBackInInventory(new ItemStack(ModItems.SHADOW.get()));
                     }
-                    attackerPlayer.getInventory().placeItemBackInInventory(new ItemStack(ModItems.SHADOW.get()));
+                } else {
+                    if (!target.getPersistentData().getBoolean("mineminenomi.ShadowStolen")) {
+                        target.getPersistentData().putBoolean("mineminenomi.ShadowStolen", true);
+                        attackerPlayer.getInventory().placeItemBackInInventory(new ItemStack(ModItems.SHADOW.get()));
+                    }
                 }
             }
